README.md

```diff
@@ -4,7 +4,6 @@ This repo is a holding area for recipes destined for a conda-forge feedstock rep
 
 [![Join the chat at https://gitter.im/conda-forge/conda-forge.github.io](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/conda-forge/conda-forge.github.io)
 
-
```

Comment:
Please don't edit other files, just the recipe you are adding.

Reply:
Oh I'll do just that. However, I still have some errors with checks after adding the required packages. I'd really appreciate it if you can point me in the right direction.